src/test/kotlin/org/openmbee/flexo/mms/LockRead.kt

```diff
@@ -9,6 +10,8 @@ class LockRead : LockAny() {
         listOf(
             "head",
             "get",
+            "patch",
```

Comment:
this is causing a test failure with 
Response should have one of the statuses: [404, 403, 405, 501] but had status 415.

Reply:
fixed